src/templates/Hero.tsx

```diff
@@ -1,50 +1,160 @@
-        title={
-          <>
-            {'The simplest way to get started with your\n'}
-            <span className="text-primary-500">home EV charging station</span>
-          </>
-        }
-        description="Get started to charge your world from your finger tips."
-        button={
-          <Link href="mailto:team@energywise.solutions?subject=I'm interested in energywise&body=I would like to learn more about energywise, please keep me updated">
-            <a>
-              <Button xl secondary>
-                Join the list
-              </Button>
+        </div>
+      </div>
+      <main className="container mx-auto px-6 pt-16 flex-1 text-center relative">
+        <div className="absolute inset-0 z-0 overflow-hidden rounded-lg">
+          <div id="hero-image" className="hero-image">
+            <motion.div
+              initial={{ opacity: 0, scale: 0.8 }}
+              animate={{ opacity: 1, scale: 1 }}
+              transition={{ duration: 1.5 }} // Adjust the duration here (in seconds)
+              className="absolute inset-0 z-0 overflow-hidden rounded-lg"
+            >
+              <Image
+                src="/assets/images/herobanner.jpg"
+                alt="Image description"
+                layout="fill"
+                objectFit="cover"
+                quality={100}
+                className="opacity-70"
+              />
+            </motion.div>
+          </div>
+        </div>
+        <div className="relative z-10">
+          <motion.h2
+            initial={{ opacity: 0, y: 50 }}
+            animate={{ opacity: 1, y: 0 }}
+            transition={{ duration: 1 }}
+            className="text-1xl md:text-2xl lg:text-4xl"
+          >
+            The simplest way to get started with your
+          </motion.h2>
+          <motion.h1
+            initial={{ opacity: 0, y: 50 }}
+            animate={{ opacity: 1, y: 0 }}
+            transition={{ duration: 1, delay: 0.5 }}
+            className="text-3xl md:text-8xl lg:text-10xl font-black uppercase mb-8"
+          >
+            home EV charging station
+          </motion.h1>
+          <div className="text-lg md:text-2xl ls:text-3xl py-2 px-4 md:py-4 md:px-10 lg:py-6 lg:px-12 bg-white bg-opacity-10 w-fit mx-auto mb-8 rounded-full survey-button">
+            <a
+              href="https://energywise.solutions/"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="block w-full h-full rounded-full flex items-center justify-center text-black font-bold text-xl md:text-2xl lg:text-4xl"
+              style={{ backgroundColor: '#E4FF1A', padding: '1rem' }}
+            >
+              Take Survey
             </a>
-          </Link>
-        }
-      />
-    </Section>
-  </Background>
-);
+          </div>
+          <form action="------" method="post" target="_blank">
+            <div className="flex flex-col md:flex-row justify-center items-center mb-4">
+              <input
```

Comment:
For the text input, use can use `onChange` property to pass on the input content. Store the user input string into a `useState` variable that the button click function will look from